bridgev2/database/ghost.go

```diff
@@ -44,8 +44,9 @@ const (
 		       name_set, avatar_set, contact_info_set, is_bot, identifiers, metadata
 		FROM ghost
 	`
-	getGhostByIDQuery = getGhostBaseQuery + `WHERE bridge_id=$1 AND id=$2`
-	insertGhostQuery  = `
+	getGhostByIDQuery       = getGhostBaseQuery + `WHERE bridge_id=$1 AND id=$2`
+	getGhostByMetadataQuery = getGhostBaseQuery + `WHERE bridge_id=$1 AND json_extract(metadata, $2)=$3`
```

Comment:
Bridges have to be compatible with postgres, so the proper syntax is `metadata->>$2`, but note that paths won't be supported